.github/workflows/build_ffmpeg.yaml

```diff
@@ -46,13 +48,12 @@ jobs:
       fail-fast: false
       matrix:
         ffmpeg-version: ["4.4.4", "5.1.4", "6.1.1", "7.0.1"]
-        runner: ["macos-m1-stable"]
     uses: pytorch/test-infra/.github/workflows/macos_job.yml@main
     with:
       job-name: Build
       upload-artifact: ffmpeg-lgpl
       repository: pytorch/torchcodec
-      runner: "${{ matrix.runner }}"
+      runner: macos-14-xlarge
```

Comment:
Q - what's the reason for changing the worker?

Reply:
To avoid the long macos-m1-stable queue. This is a GitHub hosted runner that seems to get a worker faster. I have a lot of difficulty getting a straight-forward answer to what Mac runners are arm64 versus x86, but this blog post indicates that macos-14-xlarge is arm64 (M1): https://github.blog/news-insights/product-news/introducing-the-new-apple-silicon-powered-m1-macos-larger-runner-for-github-actions/